deluge/ui/gtkui/common.py

```diff
@@ -75,7 +75,7 @@ def build_menu_radio_list(value_list, callback, pref_value=None, suffix=None, sh
         item_text = str(value)
         if suffix:
             item_text += " " + suffix
-        menuitem = gtk.RadioMenuItem(group, item_text)
+        menuitem = Gtk.RadioMenuItem(item_text, group)
```

Comment:
Ah sorry it appears in Python GTK3 they have swapped them: http://learngtk.org/tutorials/python_gtk3_tutorial/html/radiomenuitem.html